### DISCUSSION TITLE
Server-side rendering for G6 charts

### DISCUSSION THREAD

**Discussion by zalmane** [antvis/G6#3457] [Show and tell]
We created a module that uses G6 to render images in NodeJS. It uses the Canvas package for server-side rendering.
Feedback welcome (and a Star if you like it...)!
https://github.com/datayoga-io/node-g6

**Comment by hustcc** [comment]
We hope for continuous iteration to ensure quality and to upgrade to the latest version. We will include it on the official website and recommend it to everyone.
